### DISCUSSION TITLE
[Script request] Graylog

### DISCUSSION THREAD

**Discussion by zelo66** [community-scripts/ProxmoxVE#49] [Request script] [answered]
### Application Name

Graylog 

### Website

https://github.com/Graylog2/graylog2-server

### Description

 Graylog is a free and open log management platform.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
merged
